src/equations/linearized_euler_2d.jl

```diff
@@ -0,0 +1,174 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+@doc raw"""
+    LinearizedEulerEquations2D(rho_mean, v1_mean, v2_mean, c)
+        
+Linearized euler equations in two space dimensions. The equations are given by
+```math
+\partial t
+\begin{pmatrix}
+    \rho' \\ v_1' \\ v_2' \\ p'
+\end{pmatrix}
++
+\partial x
+\begin{pmatrix}
+    \bar{\rho} v_1' + \bar{v_1} /rho' \\ \bar{v_1} v_1' + \frac{p'}{\bar{\rho}} \\ \bar{v_1} v_2' \\ \bar{v_1} p' + c^2 \bar{\rho} v_1'
+\end{pmatrix}
++
+\partial y
+\begin{pmatrix}
+    \bar{\rho} v_2' + \bar{v_2} /rho' \\ \bar{v_2} v_1' \\ \bar{v_2} v_2' + \frac{p'}{\bar{\rho}} \\ \bar{v_2} p' + c^2 \bar{\rho} v_2'
+\end{pmatrix}
+=
+\begin{pmatrix}
+    0 \\ 0 \\ 0 \\ 0
+\end{pmatrix}
+```
+The bar ``\bar{(\cdot)}`` indicates uniform mean flow variables and c is the speed of sound. 
+The unknowns are the acoustic velocities ``v_1', v_2'``, the pressure ``p'`` and the density ``\rho'``.
+"""
+struct LinearizedEulerEquations2D{RealT<:Real} <: AbstractLinearizedEulerEquations{2, 4}
+    rho_mean::RealT
+    v1_mean::RealT
+    v2_mean::RealT
+    c::RealT
```

Comment:
Should this be
```suggestion
    c_mean::RealT
```
For consistency?